oneview-ansible.md

```diff
@@ -12458,7 +12458,7 @@ Manage OneView Users.
  Provides an interface to manage Users. Can create, update, and delete.
 
 #### Requirements (on the host that executes the module)
-  * python >= 2.7.9
+  * python >= 3.4.2
   * hpeOneView >= 5.4.0
```

Comment:
change hpeOneView version to latest since this ansible resource only from latest package